### PR TITLE
add shellcheck plugin

### PR DESCRIPTION
**Pull request by Luizm**
Add shellcheck plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,8 @@ place for people (and asdf itself) to look for plugins.
...
 | Scala     | [mtatheonly/asdf-scala](https://github.com/mtatheonly/asdf-scala)| [![Build Status](https://travis-ci.org/mtatheonly/asdf-scala.svg?branch=master)](https://travis-ci.org/mtatheonly/asdf-scala)
 | Serf      | [Banno/asdf-hashicorp](https://github.com/Banno/asdf-hashicorp) | [![Build Status](https://travis-ci.org/Banno/asdf-hashicorp.svg?branch=master)](https://travis-ci.org/Banno/asdf-hashicorp)
-| Shfmt     | [Luizm/asdf-shfmt](https://github.com/luizm/asdf-shfmt)         | [![Build Status](https://travis-ci.org/Luizm/asdf-shfmt.svg?branch=master)](https://travis-ci.org/luizm/asdf-shfmt)
+| Shfmt     | [luizm/asdf-shfmt](https://github.com/luizm/asdf-shfmt)         | [![Build Status](https://travis-ci.org/luizm/asdf-shfmt.svg?branch=master)](https://travis-ci.org/luizm/asdf-shfmt)
+| Shellcheck | [luizm/asdf-shellcheck](https://github.com/luizm/asdf-shellcheck) | [![Build Status](https://travis-ci.org/luizm/asdf-shellcheck.svg?branch=master)](https://travis-ci.org/luizm/asdf-shellcheck)
```
Comment (Stratus3D) [inline README.md]: These plugins are alphabetically sorted. `e` comes before `f`, so I think `shellcheck` should come before `shfmt`

**Comment by Stratus3D** [comment]
I'm also not sure why there are 5 commits on this PR branch. There should be only one or two.

**Comment by Luizm** [comment]
> I'm also not sure why there are 5 commits on this PR branch. There should be only one or two.

My bad. Sure, I will do that :)

**Comment by Luizm** [comment]
@Stratus3D Done. Could you check again?

**Comment by Stratus3D** [comment]
Thanks! Looks good to me.

**Pull request merged**